assets/src/scss/blocks/Accordion/AccordionStyle.scss

```
@@ -105,8 +105,8 @@
     }
 
     .accordion-btn {
-      margin: $sp-3 10% 0 10%;
-      line-height: 2.5;
+      margin: $sp-3 0 0 0;
```

This code also works
```suggestion
      margin: $sp-3 0 0;
```